CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+For general contributing guidelines, see the [Contributing Guide on DoneJS.com][].
+
+## Developing Locally
+
+### Setup
```

Comment:
This is my opinion: I'm not against doing this for primary repos, but practically speaking when documenting 10s of small repos it is easier to remember and follow "Setup, Build, Test". For the reader, it is much easier to parse through and find the information—especially when these are just small plugin repos and not full-blown apps or the main tool. I don't see why we need "nice consistency" between 10s of small plugin repos and the main DoneJS guide. I think we need to make a distinction here, to avoid shoehorning. Of course, consistency would exist between these small repos themselves ("Setup, Build, Test"). As a contributor, I want to be able to get the info I need, and then get out without parsing verbose headers. I feel like you're wanting to add a bunch of info that doesn't need to be in these sub-primary repos. I've had my chance to explain my reasoning (thanks for that), now I will defer the final say to you! I can do it either way, but I hope you understand what I was going for here and why.